projects/netgrif-components-core/src/lib/resources/engine-endpoint/public/public-task-resource.service.ts

```diff
@@ -86,48 +85,6 @@ export class PublicTaskResourceService extends TaskResourceService {
             .pipe(map(r => this.changeType(r, 'dataGroups')));
     }
 
-    /**
-     *  Get all task data
-     *
-     *  GET
-     *
-     *  If you want to process the raw backend response use [rawGetData]{@link TaskResourceService#rawGetData} instead.
-     *
-     * @param taskId ID of the task who's data should be retrieved from the server
-     * @returns processed data groups of the given task. If the task has no data an empty array will be returned.
-     */
-    public getData(taskId: string): Observable<Array<DataGroup>> {
```

Comment:
why is this endpoint removed ?

Reply:
After updating the code, that was duplicate of the one in task-resource.service.ts.